cmd/subctl/cloud.go

```diff
@@ -64,9 +64,8 @@ func init() {
 		port.NATTDiscovery, "NAT discovery port")
 	cloudPrepareCmd.PersistentFlags().Uint16Var(&cloudPorts.Vxlan, "vxlan-port", port.IntraClusterVxLAN, "Internal VXLAN port")
 
-	cloudPorts.Metrics = append(cloudPorts.Metrics, 8080, 8081)
-
 	cloudPrepareCmd.PersistentFlags().Var(&uint16Slice{value: &cloudPorts.Metrics}, "metrics-ports", "Metrics ports")
```

Comment:
Then we can remove `cloudPorts.Metrics` altogether:

```suggestion
        ports := []uint16{}
	cloudPrepareCmd.PersistentFlags().Var(&uint16Slice{value: &ports}, "metrics-ports", "Metrics ports")
```